SF - add DOMPurify and html-sanitize to clean the HTML

HarderBetterFasterStronger opened the pull request:
copilot:all

jaibhavaya reviewed (approved):
The pipeline always fails for this, because it tries to publish and it cannot.  once we merge I can publish this.

HarderBetterFasterStronger commented:
Tried it out locally linking editor against my local (`npm link`) and adding some console.log printing the dirty HTML vs. cleaned HTML and definitely saw it filter out my dirty code throwing an alert().

I also added a dirty image tag to a table cell and saw it prune out the content of the cell.

So, it seems to work well.
